cohort_4/playbook/ACTIVITIES.md

```diff
@@ -1569,6 +1569,7 @@ https://docs.google.com/spreadsheets/d/1D8y_UB11C4QrXB9kn86-UqVQQqT9FUDqftx1AMHI
 |   8 | https://docs.google.com/a/zachlatta.com/forms/d/1bIkPi1IWIf34WXMHxivQDOqCWBIOq2SVUl_OFw2e_9c |
 |   9 | https://docs.google.com/a/zachlatta.com/forms/d/1LFVzviI9GDuxLTpOPWy9jNmImhmW0NisAMC0ocBJzIg |
 |  10 | https://docs.google.com/a/zachlatta.com/forms/d/1SAcCBYcr8qpzXjCWLlbeeilZUObBROR-EZh0WRTguLE |
+|  10+ | https://docs.google.com/a/hackmatch.com/forms/d/1C04DICTvylS_sDutVkzwe-e-MgxzABp9PGC0XPZVCkA |
```

Comment:
Can you fix the formatting of the entire table to account for `10+` being longer than the rest of the numbers?